open_academy/wizards/wizard.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models, fields, Command
+
+
+class Wizard(models.TransientModel):
+    _name = 'wizard'
```

Comment:
Just a comment for the future: try to name the files with a descriptive name, like: wizards/assign_attendee.py because the wizard is already in the 'wizards' directory.